src/Appwrite/Auth/OAuth2/Github.php

```diff
@@ -208,4 +208,15 @@ protected function getUser(string $accessToken)
 
         return $this->user;
     }
+
```

Comment:
Is this regarding createRepository? It requires OAuth token, not application token. That's why it's here. Do we want to add complexity to VCS library of whole OAuth auth process, just to have this 1 method in it?

Reply:
the GitHub adapter should be able to operate in two modes
1. As a GitHub App ( it will be initialised with an app token ) 
2. As a GitHub User ( it will be initialised with an oauth token ) 

Based on our use case, we should be able to initialise the adapter with the appropriate token. 
If not now, we have to add a backlog task to do this refactor.